test/smoke/default/configure_test.rb

```diff
@@ -8,63 +8,6 @@
 
 describe file('/opt/sonatype/nexus/etc/fabric/hazelcast.xml') do
   it { should exist }
-
-  context 'when group name is set' do
-    its('content') { should include '<name>nexus</name>' }
-  end
-
-  context 'when group password is set' do
-    its('content') { should include '<password>nexus123</password>' }
-  end
-
-  context 'when multicast is enabled' do
```

Comment:
Looks like we dropped tests for some of these? They are still in the template.

- multicast enabled
- tcp-ip disabled
- aws disabled
- iam_role
- region
- tag key/value
- hazelcast discovery disabled

Do we not need to test these?

Reply:
these tests are just removed from this place, but they are duplicated across all the other tests. for example (not exclusively):

multicast enabled is here: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L13

tcp-ip disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L29

aws disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/multicast_enabled/configure_test.rb#L25

iam_role: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L13

region: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L21

tag: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L25

discovery disabled: https://github.com/sonatype/chef-nexus-repository-manager/blob/master/test/smoke/aws_enabled/configure_test.rb#L41

Reply:
Great! Thanks!